pages/glossary/default-page-language.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Default Page Language
+key: default-page-language
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+---
+
+The default page language is the _most common language_ used in a [web page][]. If there are no _words_ in the page, the default language is undefined.
+
+The _most common language_ is determined by counting the number of _words_ in the [web page][] that are part of any of the languages in the [language subtag registry][]. The same word can be part of multiple languages. If the number of words is the same, the default language is undefined.
```

Comment:
I'm afraid this requires dictionaries (as in, "list of words") in addition to the languages.

Different dictionaries, even for the same language, do not list the exact same words, notably for new words.
For example, as of now, the French dictionary Larousse does not accept "infox" while Le Petit Robert does accept it.
https://www.larousse.fr/dictionnaires/francais/infox
https://dictionnaire.lerobert.com/definition/infox

(and Le Petit Robert only accept it in its 2020 edition, not before).
(and both of these are reference dictionaries in French French, so for the `fr_FR` language)

So, "words that are part of a language" is actually ambiguous => according to which dictionary or dictionaries?




This gets even hairier with regional differences. "armour" is a word in `en_UK` but not in `en_US`. It should be counted as a word in `en` (because the definition only cares for the language, not for the region). But then, both "armour" and "armor" should count for `en`, which requires a list of words for each of the possible subregions.

Reply:
I think the language input addresses this. Count up all the words you understand to be part of a particular language, that's the number of words. If the dictionary you may use for that is different from the one someone else uses, that's a change that is different because the input is different. I don't think that makes this definition ambiguous.

Reply:
Good point. I am convinced.